### ISSUE TITLE
disableThirdPartySharing is not the opposite of trackThirdPartySharing

### ISSUE THREAD

**Issue by fealXX** [Tealium/tealium-android-adjust-remote-command#7]
Hi!

[Here](https://github.com/Tealium/tealium-android-adjust-remote-command/blob/03e5b3a4a5d052399c6534ad2bb98427030f8105/adjust/src/main/java/com/tealium/remotecommands/adjust/AdjustInstance.kt#L233) the function disableThirdPartySharing is used to show an opt-out, but trackThirdPartySharing as the opt-in.
Those seem to not work in the same way (disableThirdPartySharing is not doing anything obvious to me, while trackThirdPartySharing sends a request to Adjusts Endpoint for ThirdPartySharing) and I believe this implementation should be changed to always pass the enabled parameter to the trackThirdPartySharing function. This would also reflect the [Implementation in iOS](https://github.com/Tealium/tealium-ios-adjust-remote-command/blob/db7938fcbf89536234449e802a2aab1dcd2c07c5/Sources/AdjustInstance.swift#L101).

```
override fun setThirdPartySharing(enabled: Boolean) {
    val sharing = AdjustThirdPartySharing(enabled)
    //  sharing.addGranularOption()
    // TODO (how best to map granular options)
    Adjust.trackThirdPartySharing(sharing)
}
```

All the best,

Felix

**Comment by craigrouse** [comment]
Thanks for bringing this to our attention @fealXX. We'll get it updated.

**Comment by jameskeith** [comment]
@fealXX - thanks for opening the pull request. I've merged that in alongside all the relevant version updates and tests etc.
So this should now be available on our maven repository as version `1.2.0` - let us know if you still have any issues.
